setup.py

```diff
@@ -86,7 +86,11 @@
     ],
     tests_require=tests_require,
     setup_requires=["setuptools", "appdirs", "pytest-runner"],
-    extras_require=dict(test=tests_require, dask=["dask[complete]"]),
+    extras_require={
+        "test": tests_require,
+        "dask": ["dask[complete]"],
+        "profet": ["emukit", "GPy", "torch", "pybnn"],
```

Comment:
Is the default pybnn install working or we need to install a special version from their git repo?

Reply:
I think the normal version should be fine. There was an issue with the DNGO sampler, but that's taken care of in the `orion.algo.robo` plugin.

Reply:
Great!

Reply:
Is GPy needed or that was for other parts of the warm-start experiment code? I don't see it imported anywhere.